pombola/hansard/management/commands/hansard_check_for_new_sources.py

```diff
@@ -66,9 +72,16 @@ def process_url(self, list_page, url, verbose):
             convertEntities=BeautifulStoneSoup.HTML_ENTITIES
         )
 
-        spans = soup.findAll( 'span', 'contenttype-repositoryitem summary')
+        base_url = '/'.join(url.split('/')[:3])
+
+        next_url = self.get_next_url(soup, base_url)
+        get_next_page = False
 
-        links = [ span.a for span in spans ]
+        # Grab each of the blocks on the current page which are supposed to
+        # contain a link to a PDF file.
+        link_sections = soup.findAll( 'div', 'catItemView groupPrimary' )
+
+        links = [ section.find( 'div', 'catItemAttachmentsBlock' ).a for section in link_sections ]
```

Comment:
Some minor PEP 8 violations here (which the code you're replacing had too, but I think it's worth changing) - there shouldn't be a space after an opening `[` or `(` or before a closing `]` or `)`, and a few of these lines are over 80 characters; I'd probably split this one as:

```
links = [
    section.find( 'div', 'catItemAttachmentsBlock' ).a
    for section in link_sections
]
```